frontend/src/app/assignments/[id]/page.tsx

```diff
@@ -79,6 +83,39 @@ export default function Page({ params }: Props) {
       );
   };
 
+  interface PostFeedbackBody {
+    language: string;
+    sourceCode: string;
+    questionId: string;
+    studentId: number;
+  }
+
+  const handleSubmitCode = (fileContent: string) => {
+    const requestBody: PostFeedbackBody = {
+      language: "python", // need to change
+      sourceCode: fileContent,
+      questionId: "your_question_id_here", // need to change
+      studentId: userId,
+    };
+    GradingService.postFeedback(requestBody)
+      .then(() => {
+        toast({
+          title: "Code uploaded successfully",
+          description:
+            "Code uploaded successfully. Feedback will be available shortly.",
+          variant: "success",
+        });
+      })
+      .catch((_err) => {
+        toast({
+          title: "Failed to upload code",
+          description:
+            "An error occurred while uploading code. Please try again.",
+          variant: "destructive",
+        });
+      });
+  };
```

Comment:
Can try wrapping this portion in a if(fileContent) block here so that it doesn't try to submit an empty file when you first click on it.